src/linux/common/helpers/alar/base.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+# Variables
+#export UBUNTU_DISTRO="true"
+export isRedHat="false"
+export isRedHat6="false"
+export isSuse="false"
+export isUbuntu="false"
+export tmp_dir=""
+export recover_action=""
+export boot_part=""
+export rescue_root=""
+export isExt4="false"
+export isExt3="false"
+export isXFS="false"
+export isLVM="false"
+export efi_part=""
+export osNotSupported="true" # set to true by default, gets changed to false if this is the case
+export tmp_dir=""
+export global_error="false"
+STATUS_SUCCESS=0
```

Comment:
why do these have to be redefined here?

Reply:
the STATUS_SUCCESS got removed. The init.sh is sourced instead